### DISCUSSION TITLE
Proposal: <podcast:sponsored> used to identify episodes or segments that are sponsored by a particular advertiser or brand

### DISCUSSION THREAD

**Discussion by baditaflorin** [Podcastindex-org/podcast-namespace#463] [Tag Proposal]
<podcast:sponsored>: This tag could be used to identify episodes or segments that are sponsored by a particular advertiser or brand. This could allow advertisers to track the performance and effectiveness of their sponsored content, and could help data scientists to analyze the impact of sponsored content on listener engagement and behavior.

**Comment by theDanielJLewis** [comment]
I try to look at everything from the perspective of "How does this benefit the audience?"

I imagine this could be used, like the disclosure feature on YouTube videos, to indicate to the audience that the episode has some kind of "paid" placement.

This tag could potentially even be used to help disclose affiliates, as is required in many places.
